includes/defaults.inc.php

```diff
@@ -199,6 +199,9 @@
 
 // Autodiscovery Settings
 $config['autodiscovery']['xdp'] = true;
+$config['autodiscovery']['xdp_exclude']['sysdesc_regexp'][] = '/-K9W8-/'; // Cisco Lightweight Access Point
+$config['autodiscovery']['cdp_exclude']['platform_regexp'][] = '/^Cisco IP Phone/'; //Cisco IP Phone
```

Comment:
sysname and sysdesc are sent in both LLDP and CDP protocols so xdp was used for those two.

platform is sent only in the CDP protocol and has no corresponding entry in the LLDP protocol.

Here's what the same device looks like in both protocols:

```
[cdpCacheAddressType] => ip
[cdpCacheAddress] => 0A 0C 08 5A
[cdpCacheVersion] => Cisco IOS Software, C2600 Software (AP3G2-K9W8-M), Version 15.3(3)JA7, RELEASE SOFTWARE (fc2)
[cdpCacheDeviceId] => abc-102-ap1
[cdpCacheDevicePort] => GigabitEthernet0
[cdpCachePlatform] => cisco AIR-CAP2602I-A-K9
[cdpCacheCapabilities] => 00 00 00 26
[cdpCacheNativeVLAN] => 0
[cdpCacheDuplex] => fullduplex
[cdpCacheLastChange] => 79:10:46:04.79

[lldpRemChassisIdSubtype] => macAddress
[lldpRemChassisId] => 24 01 C7 F7 43 60
[lldpRemPortIdSubtype] => interfaceName
[lldpRemPortId] => Gi0
[lldpRemPortDesc] => GigabitEthernet0
[lldpRemSysName] => abc-102-ap1.xyz.k12.va.us
[lldpRemSysDesc] => Cisco IOS Software, C2600 Software (AP3G2-K9W8-M), Version 15.3(3)JA7, RELEASE SOFTWARE (fc2)
[lldpRemSysCapSupported] => 20 00
[lldpRemSysCapEnabled] => 20 00
[lldpRemManAddrIfSubtype] => systemPortNumber
[lldpRemManAddrIfId] => 0
[lldpRemManAddrOID] => zeroDotZero

```